common/src/main/java/io/novafoundation/nova/common/di/CommonApi.kt

```diff
@@ -64,6 +66,36 @@ import java.util.Random
 
 interface CommonApi {
 
+    val systemCallExecutor: SystemCallExecutor
+
+    val actionAwaitableMixinFactory: ActionAwaitableMixin.Factory
+
+    val resourcesHintsMixinFactory: ResourcesHintsMixinFactory
+
+    val okHttpClient: OkHttpClient
+
+    val fileCache: FileCache
+
+    val permissionsAskerFactory: PermissionsAskerFactory
+
+    val bluetoothManager: BluetoothManager
+
+    val locationManager: LocationManager
+
+    val listChooserMixinFactory: ListChooserMixin.Factory
+
+    val partialRetriableMixinFactory: PartialRetriableMixin.Factory
+
+    val automaticInteractionGate: AutomaticInteractionGate
+
+    val bannerVisibilityRepository: BannerVisibilityRepository
+
+    val provideActivityIntentProvider: ActivityIntentProvider
+
+    val googleApiAvailabilityProvider: GoogleApiAvailabilityProvider
+
+    val coinGeckoLinkParser: CoinGeckoLinkParser
+
```

Comment:
I dont mind this reordering but just of curiosity: what was the reason for doing that?

Reply:
I just refactored it to have fields above functions